### PR TITLE
Add user stories

### PR DESCRIPTION
**Pull request by ilyavoronin**

### CODE REVIEW

File: 8.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Как зарегистрированный пользователь, 
+я хочу контролировать какие документы, которыми со мной поделились, будут отображаться в "моих документах": от всех пользователей или только от списка моих знакомых,
```
Comment (NekoNe) [inline 8.md]: Could you define "контролировать", please?

File: 2.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+Как пользователь,
```
Comment (NekoNe) [inline 2.md]: I think there two stories in this one. The first one about sharing and the second one about giving permissions.

File: 3.md [hunk 3]
```diff
@@ -0,0 +1,4 @@
+Как зарегистрированный пользователь,
```
Comment (NekoNe) [inline 3.md]: Does "have access" mean "be able to get a list of all documents"?

File: 4.md [hunk 4]
```diff
@@ -0,0 +1,4 @@
+Как зарегистрированный пользователь,
```
Comment (NekoNe) [inline 4.md]: There are too many things in this story. Make more smaller stories, please.

File: 5.5.md [hunk 5]
```diff
@@ -0,0 +1,3 @@
+Как пользователь имеющий абсолютные права доступа к документу,
```
Comment (NekoNe) [inline 5.5.md]: What is "absolute"? It was not defined earlier.

**Review by NekoNe — CHANGES_REQUESTED** [review]

File: 1.md [hunk 6]
```diff
@@ -0,0 +1,3 @@
+Как незарегистрируемый пользователь,
```
Comment (M0nsteR) [inline 1.md]: Are you sure this is acceptable Russian grammar?

File: 1.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+Как незарегистрируемый пользователь,  
+я хочу иметь возможность зарегистрироваться с использованием уникального логина, 
+чтобы другие пользователи могли найти и идентифицировать меня.
```
Comment (M0nsteR) [inline 1.md]: I'm not sure about this motivation part. Can you think of any other reason to have a unique account?

File: 5.md [hunk 8]
```diff
@@ -0,0 +1,3 @@
+Как пользователь имеющий права редактирования документа,
```
Comment (M0nsteR) [inline 5.md]: Missing comma.

File: 6.md [hunk 9]
```diff
@@ -0,0 +1,3 @@
+Как зарегистрированный пользователь,  
+я хочу иметь возможность создавать группы зарегистрированных пользователей видимые только мне,
```
Comment (M0nsteR) [inline 6.md]: Check your punctuation.

**Review by M0nsteR — CHANGES_REQUESTED** [review]
Please combine all user stories in a single file.

File: 8.md (hunk 1 above)
Comment (ilyavoronin) [reply 8.md]: It means "to choose". "To choose which documents will be shown...". I don't know what can I change here

File: 2.md (hunk 2 above)
Comment (ilyavoronin) [reply 2.md]: Splitted into three stories: 2,3,6

File: 3.md (hunk 3 above)
Comment (ilyavoronin) [reply 3.md]: yes(fixed)

File: 4.md (hunk 4 above)
Comment (ilyavoronin) [reply 4.md]: Splitted into two stories: 6, 7

File: 5.5.md (hunk 5 above)
Comment (ilyavoronin) [reply 5.5.md]: This is the definition of this access group. "absolute" is just a name

File: 1.md (hunk 6 above)
Comment (ilyavoronin) [reply 1.md]: no i am not

File: 1.md (hunk 7 above)
Comment (ilyavoronin) [reply 1.md]: I've added another motivation

File: 5.md (hunk 8 above)
Comment (ilyavoronin) [reply 5.md]: fixed

File: 6.md (hunk 9 above)
Comment (ilyavoronin) [reply 6.md]: fixed

**Review by M0nsteR — APPROVED** [review]

**Pull request merged**